lib/loaders/loaders_with_authentication.js

```diff
@@ -0,0 +1,29 @@
+import { gravityLoaderWithAuthenticationFactory, impulseLoaderWithAuthenticationFactory } from "./api"
+const { IMPULSE_APPLICATION_ID } = process.env
+
+export default (accessToken, userID) => {
+  let impulseTokenLoader
+  const gravityLoader = gravityLoaderWithAuthenticationFactory(() => Promise.resolve(accessToken))
+  const impulseLoader = impulseLoaderWithAuthenticationFactory(() => impulseTokenLoader().then(data => data.token))
```

Comment:
It seems weird that you're using `impulseTokenLoader()` in the var declaration when it's just a `let` at the top, and only re-declared below, maybe switch the order of lines here? #minor